db/migrate/20211218160303_remove_not_null_on_active_storage_blobs_checksum.active_storage.rb

```diff
@@ -0,0 +1,6 @@
+# This migration comes from active_storage (originally 20211119233751)
+class RemoveNotNullOnActiveStorageBlobsChecksum < ActiveRecord::Migration[7.0]
+  def change
+    change_column_null(:active_storage_blobs, :checksum, true)
+  end
+end
```

Comment:
I'd like to change this to specific the algorithm and lock modes to avoid locking the db. I think it's this:

```
execute("ALTER TABLE active_storage_blobs MODIFY COLUMN checksum VARCHAR(255) NULL, ALGORITHM=INPLACE, LOCK=NONE")
```

Reply:
I've done this.